src/main/java/ysoserial/payloads/util/PayloadRunner.java

```diff
@@ -1,40 +1,41 @@
 package ysoserial.payloads.util;
 
-import static ysoserial.payloads.util.Serializables.deserialize;
-import static ysoserial.payloads.util.Serializables.serialize;
-
 import java.util.concurrent.Callable;
 
-import ysoserial.ExecBlockingSecurityManager;
+import ysoserial.Deserializer;
+import ysoserial.Serializer;
+import static ysoserial.Deserializer.deserialize;
+import static ysoserial.Serializer.serialize;
 import ysoserial.payloads.ObjectPayload;
+import ysoserial.secmgr.ExecCheckingSecurityManager;
 
 /*
  * utility class for running exploits locally from command line
  */
 @SuppressWarnings("unused")
 public class PayloadRunner {
-	public static void run(final Class<? extends ObjectPayload<?>> clazz, final String[] args) throws Exception {		
+	public static void run(final Class<? extends ObjectPayload<?>> clazz, final String[] args) throws Exception {
 		// ensure payload generation doesn't throw an exception
-		byte[] serialized = ExecBlockingSecurityManager.wrap(new Callable<byte[]>(){
+		byte[] serialized = new ExecCheckingSecurityManager().wrap(new Callable<byte[]>(){
 			public byte[] call() throws Exception {
 				final String command = args.length > 0 && args[0] != null ? args[0] : "calc.exe";
-				
+
 				System.out.println("generating payload object(s) for command: '" + command + "'");
-				
-				final Object objBefore = clazz.newInstance().getObject(command);
-				
+
+				final Object objBefore = clazz.newInstance().getObjectExec(command);
+
 				System.out.println("serializing payload");
-				
-				return serialize(objBefore);
-		}});			
-			
-		try {	
-			System.out.println("deserializing payload");			
-			final Object objAfter = deserialize(serialized);			
+
+				return Serializer.serialize(objBefore);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Serializer.serialize' due to existing static import 'ysoserial.Serializer.serialize'](https://www.codacy.com/app/frohoff/ysoserial/file/2167893687/issues/source?bid=3161887&fileBranchId=3161890#l29)